### ISSUE TITLE
RECORDINGS_METRICS_READ_FROM_DISK does not handle "No such file or directory"

### ISSUE THREAD

**Issue by 1977er** [greenstatic/bigbluebutton-exporter#44]
On fresh installations of BBB some of the directories below /var/bigbluebutton/recording do not yet exist (ie. status/published in my case).

The RECORDINGS_METRICS_READ_FROM_DISK feature cannot handle this and is running into "No such file or directory" errors.

```
bbb-exporter | 2020-07-24 10:04:51,321 [INFO]: Enabling recordings metrics read from disk, we will not request expensive recordings metrics via the API
bbb-exporter | 2020-07-24 10:04:51,322 [INFO]: HTTP server started on port: 9688
bbb-exporter | 2020-07-24 10:04:51,322 [INFO]: Collecting metrics from BigBlueButton API
bbb-exporter | Traceback (most recent call last):
bbb-exporter | File "server.py", line 46, in <module>
bbb-exporter | REGISTRY.register(collector)
bbb-exporter | File "/usr/local/lib/python3.7/site-packages/prometheus_client/registry.py", line 24, in register
bbb-exporter | names = self._get_names(collector)
bbb-exporter | File "/usr/local/lib/python3.7/site-packages/prometheus_client/registry.py", line 64, in _get_names
bbb-exporter | for metric in desc_func():
bbb-exporter | File "/app/collector.py", line 81, in collect
bbb-exporter | yield self.metric_recordings_deleted_from_disk()
bbb-exporter | File "/app/collector.py", line 261, in metric_recordings_deleted_from_disk
bbb-exporter | metric.add_metric([], recordings_deleted_from_disk(self.recordings_metrics_base_dir))
bbb-exporter | File "/app/collector.py", line 300, in recordings_deleted_from_disk
bbb-exporter | return len(os.listdir(path=os.path.join(bigbluebutton_base_dir, "recording/status/published"))) - \
bbb-exporter | FileNotFoundError: [Errno 2] No such file or directory: '/var/bigbluebutton/recording/status/published' 
```

**Comment by greenstatic** [comment]
I was afraid of that. Thanks for the report, I'll make a bugfix release later in the evening where if the directory doesn't exist it won't trigger an exception.

**Comment by 1977er** [comment]
No rush. I'm fine with RECORDINGS_METRICS_READ_FROM_DISK turned off. Just stumpled upon it. Thanks for the fast reply.

**Comment by greenstatic** [comment]
Should be solved in [release v0.5.1](https://github.com/greenstatic/bigbluebutton-exporter/releases/tag/v0.5.1). Upgrade is recommended and to re-enable `RECORDINGS_METRICS_READ_FROM_DISK`.
